gatsby-node.js

```diff
@@ -40,3 +45,62 @@ exports.onCreateWebpackConfig = ({ stage, actions, getConfig }) => {
     actions.replaceWebpackConfig(config)
   }
 }
+
+const checkIfDirExists = async folderPath => {
+  try {
+    await stat(folderPath)
+    return true
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      return false
+    }
+  }
+}
+
+const mkDirIfItDoesNotExist = async folderPath => {
+  const doesDirExist = await checkIfDirExists(folderPath)
+  if (!doesDirExist) {
+    await mkdir(folderPath, { recursive: true })
+  }
+}
+
+exports.onPostBuild = async ({ graphql }) => {
+  try {
+    const {
+      data: {
+        allBlogPost: { nodes }
+      }
+    } = await graphql(`
+      query GetBlogMainImages {
+        allBlogPost {
+          nodes {
+            picture {
+              fields {
+                sourcePath
+              }
+            }
+            date
+          }
+        }
+      }
+    `)
+
+    await Promise.all(
+      nodes.map(async node => {
+        const imagePath = node.picture.fields.sourcePath
+        const dirPath = path.dirname(
+          path.join(__dirname, 'public', 'blog', imagePath)
+        )
+        await mkDirIfItDoesNotExist(dirPath)
+        return sharp(path.join(__dirname, 'static', 'uploads', imagePath))
+          .resize({ width: BLOG.mageMaxWidthHero })
```

Comment:
Just noticed that `sharp` isn't actually resizing the images. I'll look into into what's wrong 🤔

Reply:
Fixed! It's literally just a typo, `BLOG.mageMaxWidthHero` instead of `BLOG.imageMaxWidthHero` 😆